src/lib.rs

```diff
@@ -418,6 +369,74 @@ impl Game {
         Ok(())
     }
 
+    fn kill_player_near(&mut self, position: Position) -> Result<(), JsValue> {
+        let local_player_color = match &self.local_player_color {
+            None => return Ok(()), // not controlling anything
+            Some(c) => *c,
+        };
+
+        let mut killed_player: Option<&mut Player> = None;
+        let mut closest_distance = self.kill_distance;
+
+        for player in self.players.iter_mut() {
+            // TODO(can't kill impostors)
+            if player.color == local_player_color || player.dead {
```

Comment:
I wonder if players could support an `is_local` field 🤔

Reply:
In a PR a few down the line I'm switching to UUIDs for identifying players